content/newsletters/_2024.json

```
@@ -30,6 +25,31 @@
       "month": 5,
       "file": "/images/newsletter-uploads/2024/_2024_05__Announcing the SSW Clean Architecture Template__.html",
       "description": "Announcing the SSW Clean Architecture Template!"
+    },
+    {
+      "month": 5,
+      "file": "/images/newsletter-uploads/2024/_2024_05__SSW purchases TinaCMS – the No 1 CMS on GitHub – we love Markdown__.html",
+      "description": "SSW purchases TinaCMS – the #1 CMS on GitHub – we love Markdown ❤️"
+    },
+    {
+      "month": 6,
+      "file": "/images/newsletter-uploads/2024/_2024_06__Up and running with TinaCMS in 15 minutes - No prob-llamas__.html",
+      "description": "Up and running with 🦙TinaCMS in 15 minutes - No prob-llamas!"
+    },
+    {
+      "month": 7,
+      "file": "/images/newsletter-uploads/2024/_2024_07__Showcasing TinaCMS 🦙 at Europe's No1 Developer Conference__.html",
+      "description": "Showcasing TinaCMS 🦙 at Europe's #1 Developer Conference"
+    },
+    {
+      "month": 8,
+      "file": "/images/newsletter-uploads/2024/_2024_08__How to Modernize an Enterprise Application to dotnet 8 and Lose 75 of the Code__.html",
```

```suggestion
      "file": "/images/newsletter-uploads/2024/_2024_08__How to Modernize an Enterprise Application to dotnet 8 and Lose 75 Percent of the Code__.html",
```

In line with my other suggestion